### ISSUE TITLE
Add warning if no results found by `createMergedResultsFile`

### ISSUE THREAD

**Issue by MaximMoinat** [OHDSI/CohortDiagnostics#1045]
## Description
If there is no results zip in the given `dataFolder` when running `createMergedResultsFile`, it will show that it connects to a database and then exits (see screenshot). This does create a `.sqlite` file with all necessary results tables, but no records are loaded. Then, when running the Shiny app, I just get the error 'Object databases not found'. This is especially confusing when the result csv's are in the given folder, but for some reason there is no zip of the results.

<img width="1218" alt="image" src="https://user-images.githubusercontent.com/17825660/230585004-0cf3f416-23b8-4226-9854-e00785024fc3.png">
(Using version 3.1.0)

## Suggested fix
Display a warning "No result zip files found", on line 197 here:
https://github.com/OHDSI/CohortDiagnostics/blob/0e3764fd7a223838315d9ef2d9cd6cb7f5772870/R/Shiny.R#L197-L212

**Comment by azimov** [comment]
Agreed - but i think stopping/throwing an error is better as it will avoid leaving an empty database file in the folder
